### ISSUE TITLE
Dock gets stuck if repeatedly triggered with animations disabled

### ISSUE THREAD

**Issue by micheleg** [micheleg/dash-to-dock#539]
Reported by Marsal via mail.

With animations disabled (tweaks-tool to animation-duration set to 0) the docks sometimes gets stuck hidden after repeatedly triggering in succession the dock autohide.

**Comment by jtojnar** [comment]
This is quite annoying. Any ideas what could be causing it? I am able to reproduce it even with extension settings reset:

```
$ dconf dump /org/gnome/desktop/interface/
[/]
enable-animations=false

$ dconf dump /org/gnome/shell/extensions/dash-to-dock/
```

**Comment by micheleg** [comment]
Should be fixed now in master [13c2efc].
